core/state_transition.go

```diff
@@ -317,23 +317,24 @@ func (st *StateTransition) TransitionDb() (*ExecutionResult, error) {
 		return nil, fmt.Errorf("%w: have %d, want %d", ErrIntrinsicGas, st.gas, gas)
 	}
 	if st.evm.ChainConfig().IsCancun(st.evm.Context.BlockNumber) {
-		var targetBalance, targetNonce, targetCodeSize, targetCodeKeccak, originBalance, originNonce []byte
+		var targetBalance, targetNonce, targetCodeSize, targetCodeKeccak, originBalance, originNonceBytes []byte
 
 		targetAddr := msg.To()
 		originAddr := msg.From()
 
 		statelessGasOrigin := st.evm.Accesses.TouchTxOriginAndComputeGas(originAddr.Bytes(), msg.Value().Sign() != 0)
 		if !tryConsumeGas(&st.gas, statelessGasOrigin) {
-			return nil, fmt.Errorf("insufficient gas to cover witness access costs")
+			return nil, fmt.Errorf("insufficient gas to cover witness access costs for tx")
```

Comment:
there's no error parameter here, so might as well declare it as `errors.New`, and maybe move it out in its own variable, because that same error is returned line 337.